nixos/modules/system/boot/tmp.nix

```diff
@@ -2,6 +2,9 @@
 
 with lib;
 
+let
```

Comment:
not sure if a local variable is justified for just one use?

Reply:
now theres 2